build.sbt

```diff
@@ -2,7 +2,7 @@ import com.typesafe.tools.mima.core._
 
 Global / onChangedBuildSource := ReloadOnSourceChanges
 
-ThisBuild / tlBaseVersion := "3.4"
+ThisBuild / tlBaseVersion := "3.5"
```

Comment:
This change is source-breaking: `stdin` now requires an `Async` constraint. So we may want to sit on this PR until we are ready for the minor bump.